src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -385,6 +372,35 @@ export class DotEditContentHtmlService {
         return this.getEditPageIframe().contentDocument || this.getEditPageIframe().contentWindow.document;
     }
 
+    private handlerContentletEvents(event: string): (contentletEvent: any) => void {
+        const contentletEventsMap = {
+            // When an user create or edit a contentlet from the jsp
+            save: (contentletEvent: any) => {
+                if (this.currentAction === Action.ADD) {
+                    this.renderAddedContentlet(contentletEvent.data);
+                } else if (this.currentAction === Action.EDIT) {
+                    this.renderEditedContentlet(contentletEvent.data);
+                }
+                this.iframeActions.next({
```

Comment:
After save and select we send the "close" event to the iframe actions that the angular component is subscribe to.